xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp

```diff
@@ -513,6 +502,17 @@ bool CAESinkPipewire::Initialize(AEAudioFormat& format, std::string& device)
     return false;
   }
 
+  CLog::Log(LOGDEBUG, "CAESinkPipewire::{} - rate: {}", __FUNCTION__, format.m_sampleRate);
```

Comment:
First entry not correct for EAC3 - but minor nitpick again

Reply:
I think we resolved this.